packages/frontend-shared/src/gql-components/topnav/LoginModal.vue

```diff
@@ -103,18 +129,52 @@ fragment LoginModal on Query {
 }
 `
 
+gql`
+mutation LoginModal_ResetAuthState {
+  resetAuthState {
+    ...Auth
+  }
+}
+`
+
+const resetAuth = useMutation(LoginModal_ResetAuthStateDocument)
+
 const setIsOpen = (value: boolean) => {
   emit('update:modelValue', value)
+  if (!value) {
+    resetAuth.executeMutation({})
+  }
 }
 const { t } = useI18n()
 
 const viewer = computed(() => props.gql?.cloudViewer)
+const errorType = computed(() => {
+  const { name } = props.gql.authState
+
+  if (name === 'AUTH_COULD_NOT_LAUNCH_BROWSER') {
```

Comment:
It looks like `AUTH_COULD_NOT_LAUNCH_BROWSER` and `AUTH_ERROR_DURING_LOGIN` map 1:1 to `browserError` and `loginError` respectively. Is there any benefit to adding what is effectively a type alias, instead of just using the string constants, `AUTH_COULD_NOT_LAUNCH_BROWSER` and `AUTH_ERROR_DURING_LOGIN`, as is?